### ISSUE TITLE
Working with Prokaryotic genomes

### ISSUE THREAD

**Issue by monicacecilia** [GMOD/Apollo#868]
## Prokaryotes
This entry was created as part of issue #866 by @erasche on 2016-02-11. I split the issue to address separately. 

===
[A] list of things I promised my PI I'd file as issues, that are mostly prokaryote specific things:

- [ ] We don't use mRNAs, *at all*. They're not biologically relevant to us and I'd like to investigate making that an optional (default on) feature for individual organisms.
- [ ] Non-canonical splice sites aren't useful to us, but that may already be configurable, I can't remember.
- [ ] #207 
-  you can annotate tRNAs right? Did I just disable some config option there?
- [ ] it would be nice to have a vertical split between sense and antisense features.
- [ ] clicking a feature could highlight the associated protein coding region. The "coloured frames" option is nice, but they would prefer to see a coloured box overlaying the protein sequence, in the correct translation frame.
- [ ] Protein translations shown in jbrowse popups on clicking feature from feature tracks (but that may be more of a JBrowse issue)
- We're still figuring out how we're going to handle the org vs chromosome thing ... we think we're going to abuse the multiple chromosome feature to allow us to store multiple genome assemblies when that occurs, so it's good that the generic label "Reference Sequence" is used everywhere.

**Comment by nathandunn** [comment]
@erasche  Definitely  more discussion here.   I DO know that you CAN annotate tRNA's, though:

<img width="318" alt="screen shot 2016-02-15 at 10 29 12 am" src="https://cloud.githubusercontent.com/assets/751274/13057161/245cdde4-d3cf-11e5-99b8-9dc30b5cbf14.png">

Not sure what was up with your instance.

**Comment by cmdcolin** [comment]
@nathandunn  Looks like tRNA was is missing from the WebApolloCanvasFeatures which I think is what @erasche uses (ehem...prolly my oversight)

No reason not to add it back though

**Comment by nathandunn** [comment]
Yes, definitely add it back, thanks.

**Comment by erasche** [comment]
@cmdcolin yep, we love canvas features a bit too much :) That may very well explain it.
